File: app/src/main/java/com/akoufatzis/weatherappclean/utils/BindingAdapters.java
package com.akoufatzis.weatherappclean.utils;

import android.databinding.BindingAdapter;
import android.widget.TextView;

/**
 * Created by alexk on 11.05.17.
 */

public class BindingAdapters {

    @BindingAdapter("android:drawableEnd")
    public static void drawableEnd(TextView view, int resourceId) {
        view.setCompoundDrawablesWithIntrinsicBounds(0, 0, resourceId, 0);
    }
}

File: app/src/main/java/com/akoufatzis/coolweather/presentation/core/BindingAdapters.kt
package com.akoufatzis.coolweather.presentation.core

import android.widget.ImageView
import androidx.databinding.BindingAdapter

object BindingAdapters {

    @JvmStatic
    @BindingAdapter("android:src")
    fun setImageResource(imageView: ImageView, resource: Int) {
        imageView.setImageResource(resource)
    }
}
